ENIGMAsystem/SHELL/Universal_System/object.cpp

```diff
@@ -41,6 +41,7 @@ namespace enigma
 
     #ifdef DEBUG_MODE
       static inline int DEBUG_ID_CHECK(int id, int objind) {
+        if (id < 0) return -1;
```

Comment:
This is actually normal because the global object has a negative id, hence the cause of this false positive. Josh said to make the macro check if we're in the initialization.